pkg/systemd/overview-cards/cryptoPolicies.jsx

```diff
@@ -160,51 +188,132 @@ const CryptoPolicyDialog = ({
                footer={
                    <>
                        {error && <ModalError dialogError={typeof error == 'string' ? error : error.message} />}
-                       <Button id="crypto-policy-save-reboot" variant='primary' onClick={() => setPolicy(true)}>
+                       {inProgress &&
+                           <Flex spaceItems={{ default: 'spaceItemsSm' }} alignItems={{ default: 'alignItemsCenter' }} className="fips-mode-progress">
+                               <Spinner isSVG size="md" />
```

Comment:
Hmm, I thought an `isLoading` button already showed a spinner.. we don't want two of them.

Reply:
Current version still has two spinners:

![image](https://user-images.githubusercontent.com/200109/161253468-bff6fc54-4687-4405-80d5-d5064978263c.png)

This is broken -- they even rotate in opposite directions! :fearful: